src/bp/core/services/ghost/service.ts

```diff
@@ -215,6 +238,10 @@ export class ScopedGhostService {
     await this.primaryDriver.upsertFile(fileName, content, recordRevision)
     this.events.emit('changed', fileName)
     await this._invalidateFile(fileName)
+
+    if (syncDbToDisk) {
+      await this.cache.sync(JSON.stringify({ rootFolder, botId: this.botId }))
```

Comment:
why not leave the responsibility of serializing the obj do the sync method ?

Reply:
Trying to keep the logic in redis cache minimal